implementations/rust/ockam/ockam/src/entity/remote.rs

```diff
@@ -0,0 +1,24 @@
+use crate::Route;
+
+use ockam_core::lib::Display;
+use serde::{Deserialize, Serialize};
+use std::fmt::{Formatter, Result};
+
+#[derive(Serialize, Deserialize, Debug, Clone, Hash, Ord, PartialOrd, Eq, PartialEq)]
+pub struct RemoteEntity {
+    pub route: Route,
+}
+
+impl Display for RemoteEntity {
+    fn fmt(&self, f: &mut Formatter<'_>) -> Result {
+        self.route.fmt(f)
+    }
+}
+
+impl RemoteEntity {
+    pub fn create<R: Into<Route>>(route: R) -> Self {
```

Comment:
I'm sure we inconsistently use new vs create around the codebase, but don't remember what is the right choice here

Reply:
This one should probably be `new`. I think the distinction we are using is `new` are always simple constructors and `create` can take high level actions such as starting workers. Which this is not